fotoobo/helpers/config.py

```diff
@@ -71,15 +73,24 @@ def load_configuration(self, config_file: Union[Path, None]) -> None:
                 if not self.inventory_file.is_absolute():
                     self.inventory_file = config_file.parent / self.inventory_file
 
-                if loaded_config.get("logging"):
-                    self.logging = loaded_config["logging"]
-
-                if loaded_config.get("audit_logging"):
-                    self.audit_logging = loaded_config["audit_logging"]
-
+                self.logging = loaded_config.get("logging", {})
+                self.audit_logging = loaded_config.get("audit_logging", {})
                 self.no_logo = loaded_config.get("no_logo", self.no_logo)
-
                 self.ssl_verify = loaded_config.get("ssl_verify", self.ssl_verify)
 
+                self.vault = loaded_config.get("vault", {})
+                if self.vault:
+                    if (
```

Comment:
Pls add a comment describing that this is a completeness check for the keys

Reply:
Adde the following comment:
  
    # Check if all the mandatory settings in vault section are given by substracting
    # the list of keys in vault config from list of mandatory keys.